### PR TITLE
Clean-up

### PR DESCRIPTION
**Pull request by MarcinMoskala**
Extract methods from long methods
Apply correct formatting according to:
https://kotlinlang.org/docs/reference/coding-conventions.html#class-header-formatting

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -13,7 +13,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.0'
+        classpath 'com.android.tools.build:gradle:2.3.1'
```
Comment (ghostbuster91) [inline build.gradle]: Change not related to commit neither to PR.

File: recycler/src/main/java/com/elpassion/android/commons/recycler/components/base/MutableListItemsStrategy.kt [hunk 2]
```diff
@@ -35,4 +30,5 @@ class MutableListItemsStrategy<T> : MutableItemsStrategy<T> {
     override fun remove(item: T) {
         items.remove(item)
     }
+
```
Comment (ghostbuster91) [inline recycler/src/main/java/com/elpassion/android/commons/recycler/components/base/MutableListItemsStrategy.kt]: Can't find any info in provided link about empty methods at the end of class

File: recycler-example/src/main/java/com/elpassion/android/commons/recycler_example/group/StableMutableRecyclerWithSectionsActivity.kt [hunk 3]
```diff
@@ -21,25 +22,37 @@ class StableMutableRecyclerWithSectionsActivity : AppCompatActivity() {
...
-            restoreSectionButton.isEnabled = true
-            clearSectionButton.isEnabled = false
+            onClearClicked(adapterCompositor, itemsStrategy)
         }
         restoreSectionButton.setOnClickListener {
-            itemsStrategy.addAll("A", createUsersWithASection().map(::StableUserItemAdapter))
-            adapterCompositor.notifyDataSetChanged()
-            restoreSectionButton.isEnabled = false
-            clearSectionButton.isEnabled = true
+            orRestoreClicked(adapterCompositor, itemsStrategy)
```
Comment (ghostbuster91) [inline recycler-example/src/main/java/com/elpassion/android/commons/recycler_example/group/StableMutableRecyclerWithSectionsActivity.kt:34]: Generally speaking it is ok to extract this methods but should be done in different commit.

**Review by ghostbuster91 — DISMISSED** [review]

**Comment by ghostbuster91** [comment]
The commit message should be more explanatory.

File: recycler/src/main/java/com/elpassion/android/commons/recycler/components/base/MutableListItemsStrategy.kt (hunk 2 above)
Comment (MarcinMoskala) [reply recycler/src/main/java/com/elpassion/android/commons/recycler/components/base/MutableListItemsStrategy.kt]: You mean empty line? Mistake

File: recycler-example/src/main/java/com/elpassion/android/commons/recycler_example/group/StableMutableRecyclerWithSectionsActivity.kt (hunk 3 above)
Comment (MarcinMoskala) [reply recycler-example/src/main/java/com/elpassion/android/commons/recycler_example/group/StableMutableRecyclerWithSectionsActivity.kt:34]: Do you want to revert it or is it ok?

File: build.gradle (hunk 1 above)
Comment (MarcinMoskala) [reply build.gradle]: Do you want it to be reverted?

Comment (ghost) [reply build.gradle]: @MarcinMoskala this change is not related to this pull request so reverting and adding this change as another PR (maybe there's more dependencies to update like Kotlin plugin) would be a great option.

**Review by OrdonTeam — APPROVED** [review]
